lib/browser/chrome-extension.js

```diff
@@ -315,9 +317,13 @@ app.once('ready', function () {
   const init = BrowserWindow.prototype._init
   BrowserWindow.prototype._init = function () {
     init.call(this)
-    hookWindowForTabEvents(this)
-    this.webContents.on('devtools-opened', () => {
-      loadDevToolsExtensions(this, objectValues(manifestMap))
-    })
+    exports.loadDevToolsExtensions(this.webContents)
   }
 })
+
+exports.loadDevToolsExtensions = function (webContents) {
```

Comment:
There is already a function named `loadDevToolsExtensions`, probably use a new name for this one?